test/v2/modules/addresses/controller.test.js

```diff
@@ -79,7 +79,8 @@ experiment('v2/modules/addresses/controller', () => {
       let result
 
       beforeEach(async () => {
-        payload = omit(addressData, 'id')
+        delete addressData.id
+        payload = addressData
```

Comment:
This probably wants to follow the same pattern as line 50, ie. create a payload and _then_ delete stuff from it
```suggestion
        payload = { ...addressData }
        delete payload.id
```

Obviously the tests are working without the change but it would be a little neater to do it this way around.